channeldb/graph.go

```diff
@@ -1728,6 +1742,7 @@ func (c *ChannelGraph) FilterChannelRange(startHeight, endHeight uint32) ([]uint
 	byteOrder.PutUint64(chanIDStart[:], startChanID.ToUint64())
 	byteOrder.PutUint64(chanIDEnd[:], endChanID.ToUint64())
 
+	channelsPerBlock := make(map[uint32][]lnwire.ShortChannelID)
```

Comment:
nit: since the reset method will be run before the db is queried the first time, don't have to allocate here